requirements/django.in

```diff
@@ -5,4 +5,5 @@
 
 Django>=1.8,<2.0
 django-pyfs>=1.0.5
+-e git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
```

Comment:
This pinning is unfortunate - but necessary to test XBlock at the same version that edx-platform will use. Could I request that you add a note [here](https://openedx.atlassian.net/browse/PLAT-1882) about upgrading this dependency when edx-platform moves back to using the upstream DRF version?